### ISSUE TITLE
Organization and Prioritization could be better

### ISSUE THREAD

**Issue by jLouzado** [brianlovin/security-checklist#70]
Would recommend that there be sections, related to risk tolerance and user-needs, which will help organize the recommendations better.

For example, what might be relevant for a journalist hiding out covertly in a hostile country would be very different from the average internet user.

Would be open to contributing, but I think if the average user sees a long list they're more likely to just give up entirely rather than dip their toes in and start the journey.

**Comment by mknepprath** [comment]
I had a similar thought - I was imagining maybe 4-6 visible list items that everyone should do, and then the rest (that may be unnecessary for most folks or are more "paranoid") are unlocked or expanded once those beginner items are completed. Nice suggestion! Thoughts, @brianlovin?

**Comment by brianlovin** [comment]
Yeah, I agree the list is getting a bit long and certain topics are for more advanced/paranoid folks. That being said, this list definitely isn't intended for informing a journalist hiding in a hostile country; there are dramatically different measures for someone with those stakes.

Anyways: not sure how to design/approach this long list problem yet except to be diligent in things we add. Yesterday I made it so that sections with more than 3 apps truncate to only show the first 3 apps by default. Small things like this may help reduce the initial information overload for new people.

**Comment by jLouzado** [comment]
If we have a clear idea of who exactly this page is targeting then the content can be organised around that. Do we have any traffic data on that? Or maybe, any survey of what the average level of protection the users of this site are at?

**Comment by dmleong** [comment]
> Also to be noted, technical people don't always have the best opsec.

Maybe a slider for how paranoid you are :joy: Looks like this is being discussed here https://github.com/brianlovin/security-checklist/issues/88

**Comment by mknepprath** [comment]
A slider that goes from `Beginner` to `Threat Level Midnight`. I like it.

**Comment by jgallias** [comment]
I do agree in general with making it as easy, friendly, and accessible to the widest possible audience... techie people are generally already aware or able to find and consume these resources, it's friends and family we need to be able to have them go through this list as they are able or with the right push
